src/re_frame_tracer/core.cljs

```diff
@@ -42,34 +42,35 @@
           (let [title (if protocol
                         (str protocol " " name " " arglist)
                         (str ns "/" name
+                             (when tag (str " " tag))
                              (when dispatch-val
                                (str " " (pr-str dispatch-val)))
                              (str " " arglist)
                              (when anonymous? " (anonymous)")))
                 arglist (remove '#{&} arglist)]
             (.groupCollapsed js/console "%c%s" (str "color:" color ";") title)
-            (.groupCollapsed js/console "bindings"))
-          
+            (.group js/console "bindings"))
```

Comment:
The thing is, when I open a collapsed top-level entry, 90% of the time I want to look at the bindings as well, which means that I have to click twice pretty much every single time and I'm assuming I'm not alone in this.